documentation/source/Device.md

```diff
@@ -0,0 +1,179 @@
+# Training modes
+
+SuperGradients allows users to train models on different modes:
+- CPU
+- single GPU - (CUDA)
+- single GPU - Data Parallel (DP)
+- multiple GPUs' - Distributed Data Parallel (DDP)
+
+
+### CPU
+**Requirement**: None.
+
+**How to use it**: If you don't have any CUDA device available, your training will automatically be run on CPU.
+Otherwise, the default device will be CUDA, but you can still easily set it to CPU using `setup_device` as follow:
+```py
+from super_gradients import Trainer
+from super_gradients.training.utils.distributed_training_utils import setup_device
+
+setup_device(device='cpu')
+
+# Unchanged
+trainer = Trainer(...)
+trainer.train(...)
+```
+
+
+
+### CUDA
+**Requirement**: Having at least one CUDA devices available
+
+**How to use it**: If you have at least one CUDA device, nothing! Otherwise, you will have to use CPU...
+
+
+
+### DP - Data Parallel
+**Requirement**: Having at least one CUDA devices available
+
+**How to use it**: All you need to do is to call a magic function `setup_device` before instantiating the Trainer.
+```py
+from super_gradients import Trainer
+from super_gradients.training.utils.distributed_training_utils import setup_device
+
+# Launch DP on 4 GPUs'
+setup_device(multi_gpu='DP', num_gpus=4)
+
+# Unchanged
+trainer = Trainer(...)
+trainer.train(...)
+```
+**Tip**: To optimize running time we recommend to call `setup_device` as early as possible.
+
+
+
+### DDP - Distributed Data Parallel
```

Comment:
I think a section about DDP/Devices when training with configs is important as well.

Reply:
Its pretty short: https://github.com/Deci-AI/super-gradients/blob/7e6af4cc70e0b27c7070f7e3f534a07aaa996dc6/documentation/source/Device.md?plain=1#L216-L229